CHANGES.rst

```diff
@@ -107,6 +107,8 @@ API changes
 
 - ``astropy.wcs``
 
+  - Improved documentation of WCS.calc_footprint [#3551]
```

Comment:
Far more important seems to be the change that this now raises a `ValueError` if the image dimensions can't be determined.